packages/manager/apps/pci-ai-notebooks/src/pages/notebooks/onboarding/Onboarding.page.tsx

```diff
@@ -0,0 +1,10 @@
+const Onboarding = () => {
+  return (
+    <div>
+      <h2>Hello Onboarding Page</h2>
+      <p>This is the future onboarding page</p>
+    </div>
+  );
+};
+
+export default Onboarding;
```

Comment:
I suppose this will be added in an upcoming PR

Reply:
Yes, will be done in another branch